src/app/lib/config/services/afrique.ts

```diff
@@ -326,7 +326,7 @@ export const service: DefaultServiceConfig = {
         text: 'Pourquoi vous pouvez faire confiance à BBC News',
       },
       externalLink: {
-        href: 'https://www.bbc.co.uk/editorialguidelines/guidance/feeds-and-links',
+        href: 'https://www.bbc.com/editorialguidelines/guidance/feeds-and-links',
```

Comment:
Should we change these to .com? This will probably add loads of unnecessary redirects

Reply:
Eek let me check.